.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,59 @@
+
+name: 🐞 Bug
+description: Report an issue to help improve the project.
+title: "[BUG] <titile>"
+labels: ["bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please serch to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue, also include what you tried and what didn't work
+    validations:
+      required: true
+  - type: textarea
+    id: Reproduce
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce this bug
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this bug? (like linux distribution(version + arch), python version and etc)   
+    validations:
+      required: false
+  - type: dropdown
+    id: component
+    attributes:
+      label: Where did you see this issues?
+      options:
+        - Agent
+        - Server
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: component-version
+    attributes:
+      label: In which version? 
+      description: In which version(of agent or server) this issue was found?
```

Comment:
"In which version was this issue found?"

I don't think we need to call out the agent|server here since we've got a pulldown for that... it just makes the sentence harder to parse. 😆